monitoring/accessing-third-party-monitoring-uis-and-apis.adoc

```diff
@@ -8,7 +8,6 @@ toc::[]
 
 [role="_abstract"]
 In {product-title} {product-version}, you cannot access third-party web browser user interfaces (UIs) for the following monitoring components: Alertmanager, Prometheus, Thanos Ruler, and Thanos Querier. 
-However, you can access the web UI for Grafana, although this access is deprecated and is planned to be removed in a future {product-title} release.
 In addition, you can access web service APIs for third-party monitoring components from the command line interface (CLI).
```

Comment:
All the third-party UI are disabled now, grafana is disabled for both API and UI.  Maybe better just write API only here

You can access web service API from the command line interface (CLI) for the following third-party monitoring components: : Alertmanager, Prometheus, Thanos Ruler, and Thanos Querier.

Reply:
Revised so that we no longer mention third-party web UIs at all -- only APIs for Alertmanager, Prometheus, Thanos Ruler, and Thanos Querier.